tornado-drivers/spirv/src/main/java/uk/ac/manchester/tornado/drivers/spirv/SPIRVEventPool.java

```diff
@@ -43,10 +43,9 @@ public class SPIRVEventPool {
 
     private final int poolSize;
     private final BitSet retain;
-    private int eventPositionIndex;
-
     private final HashMap<Integer, LinkedList<TimeStamp>> events;
     private final EventDescriptor[] descriptors;
+    private int eventPositionIndex;
```

Comment:
Why is this field needed?

Reply:
It used in the whole class, its just moved due to formatter

Reply:
Ok, it looked strange in the diff just one line.